packages/plugin-conversation/package.json

```diff
@@ -25,6 +25,7 @@
     "@ciscospark/test-helper-file": "^0.7.0-alpha",
     "@ciscospark/test-helper-mocha": "^0.7.0-alpha",
     "@ciscospark/test-helper-make-local-url": "^0.7.0-alpha",
+    "@ciscospark/test-helper-sinon": "^0.7.0-alpha",
```

Comment:
@ianwremmel, I noticed that this was not included, which was strange.  I found this by cleaning all the modules, and running bootstrap. 😃 I could only smile in hindsight to that one.